packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -111,36 +111,27 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
 
   public render(): JSX.Element {
     const { isStickyTop, isStickyBottom } = this.state;
+    const { stickyClassName, children } = this.props;
 
     return (
       <div ref={this._root}>
         {this.canStickyTop && (
-          <div
-            className={this.props.stickyClassName}
-            ref={this._stickyContentTop}
-            aria-hidden={!isStickyTop}
-            style={this._getStickyContainerStyle()}
-          >
+          <div ref={this._stickyContentTop} aria-hidden={!isStickyTop}>
             <div style={this._getStickyPlaceholderHeight(isStickyTop)} />
           </div>
         )}
         {this.canStickyBottom && (
-          <div
-            className={this.props.stickyClassName}
-            ref={this._stickyContentBottom}
-            aria-hidden={!isStickyBottom}
-            style={this._getStickyContainerStyle()}
-          >
+          <div ref={this._stickyContentBottom} aria-hidden={!isStickyBottom}>
             <div style={this._getStickyPlaceholderHeight(isStickyBottom)} />
           </div>
         )}
         <div style={this._getNonStickyPlaceholderHeight()} />
         <div
           ref={this._nonStickyContent}
-          className={isStickyTop || isStickyBottom ? this.props.stickyClassName : undefined}
+          className={isStickyTop || isStickyBottom ? stickyClassName : undefined}
           style={this._getContentStyles(isStickyTop || isStickyBottom)}
```

Comment:
Shouldn't we be avoiding using inline styles completely? They circumvent the `styles` API since they always take precedence... 

It looks like this wasn't converted properly when this component was converted to styles/getStyles

Reply:
not necessarily. anything that will frequently update, like a calculated top/left of a menu, should be inline. otherwise you end up building 1 class per permutation.

Reply:
but i will say it's rather confusing to have className logic in the render function when this should just be in styling.